var/spack/repos/builtin/packages/bigdft-chess/package.py

```diff
@@ -15,6 +15,7 @@ class BigdftChess(AutotoolsPackage, CudaPackage):
     git = "https://gitlab.com/l_sim/bigdft-suite.git"
 
     version("develop", branch="devel")
+    version("1.9.5", sha256="5fe51e92bb746569207295feebbcd154ce4f1b364a3981bace75c45e983b2741")
```

Comment:
Just curious if you tried compiling `1.9.4` and `1.9.3`  without any other changes, if so we could consider adding their hashes as well.
https://gitlab.com/l_sim/bigdft-suite/-/releases

Reply:
@iamashwin99 I'd like to resolve this conversation to allow merging this PR.

Issues/questions with the versions will resolve itself as they become even more obsolete.

Reply:
Sure! This was just a thought; I wanted to see if the other versions also compile with the new changes.
We can also ignore the older versions for now.